src/main/java/apoc/meta/Meta.java

```diff
@@ -430,6 +432,126 @@ private void collectStats(Collection<String> labelNames, Collection<String> relT
         return Stream.of(new MapResult(nodes));
     }
 
+
+    // Start new code
+
+    /**
+     * This procedure is intended to replicate what's in the core Neo4j product, but with the crucial difference that it
+     * supports flexible sampling options, and does not scan the entire database.  The result is producing a table of
+     * metadata that is useful for generating "Tables 4 Labels" schema designs for RDBMSs, but in a more performant way.
+     */
+    @Procedure
+    @Description("apoc.meta.nodeTypeProperties()")
+    public Stream<Tables4LabelsProfile.NodeTypePropertiesEntry> nodeTypeProperties(@Name(value = "config",defaultValue = "{}") Map<String,Object> config) {
+        MetaConfig metaConfig = new MetaConfig(config);
+        try {
+            return collectTables4LabelsProfile(metaConfig).asNodeStream();
+        } catch (Exception exc) {
+            exc.printStackTrace();
+        }
+
+        return null;
+    }
+
+    /**
+     * This procedure is intended to replicate what's in the core Neo4j product, but with the crucial difference that it
+     * supports flexible sampling options, and does not scan the entire database.  The result is producing a table of
+     * metadata that is useful for generating "Tables 4 Labels" schema designs for RDBMSs, but in a more performant way.
+     */
+    @Procedure
+    @Description("apoc.meta.relTypeProperties()")
+    public Stream<Tables4LabelsProfile.RelTypePropertiesEntry> relTypeProperties(@Name(value = "config",defaultValue = "{}") Map<String,Object> config) {
+        MetaConfig metaConfig = new MetaConfig(config);
+        try {
+            return collectTables4LabelsProfile(metaConfig).asRelStream();
+        } catch (Exception exc) {
+            exc.printStackTrace();
```

Comment:
same as above on error handling

Reply:
Done - pushed the stacktrace to debug (instead of warn) and wrapped in RuntimeException. As it's something that shouldn't really happen, I thought debug would be better than warn.